### PR TITLE
update syncer.to.checkpoint info

### PR DESCRIPTION
**Pull request by glkappe**
There is a problem with the `type` level. It is at the same level as the schema.
Optimize the document structure to make it easier to read

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by glkappe** [comment]
cc @TomShawn 

@lichunzhu PTAL

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 1]
```diff
@@ -334,22 +334,19 @@ tbl-name = "~^a.*"
 
 ### syncer.to.checkpoint
 
-以下是 `syncer.to.checkpoint` 相关的配置项。
+* `type`：指定用哪种方式保存同步进度，目前支持的选项：`mysql` 和 `tidb`。
```
Comment (lichunzhu) [inline tidb-binlog/tidb-binlog-configuration-file.md:337]:
```suggestion
* `type`：指定用哪种方式保存同步进度，目前支持的选项：`mysql`, `tidb` 和 `file`。
```

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 2]
```diff
@@ -334,22 +334,19 @@ tbl-name = "~^a.*"
...
 
-### type
+> **注意：**
+> 当 file 类型的下游 checkpoint 保存在本地文件系统
```
Comment (lichunzhu) [inline tidb-binlog/tidb-binlog-configuration-file.md:340]:
```suggestion
> file 类型的下游 checkpoint 保存在 `<data-dir>/savepoint` 文件中
```

**Review by lichunzhu — COMMENTED** [review]
Rest LGTM

**Comment by glkappe** [comment]
thanks for @lichunzhu

**Review by lichunzhu — APPROVED** [review]
LGTM

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 3]
```diff
@@ -334,22 +334,19 @@ tbl-name = "~^a.*"
 
 ### syncer.to.checkpoint
 
-以下是 `syncer.to.checkpoint` 相关的配置项。
+* `type`：指定用哪种方式保存同步进度，目前支持的选项：`mysql`, `tidb` 和 `file`。
```
Comment (TomShawn) [inline tidb-binlog/tidb-binlog-configuration-file.md:337]:
```suggestion
* `type`：指定用哪种方式保存同步进度，目前支持的选项为 `mysql`、`tidb` 和 `file`。
```

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 4]
```diff
@@ -334,22 +334,19 @@ tbl-name = "~^a.*"
...
-        > 在同个 TiDB 集群中部署多个 Drainer 时，需要为每个 Drainer 节点指定不同的 checkpoint schema，否则两个实例的同步进度会互相覆盖。
-
-    * `host`
-    * `user`
-    * `password`
-    * `port`
+* `host`
+* `user`
+* `password`
+* `port`
```
Comment (TomShawn) [inline tidb-binlog/tidb-binlog-configuration-file.md:352]: 是说这些配置项都是 syncer.to.checkpoint 下的，而不是 type 下的吗？

Comment (glkappe) [reply tidb-binlog/tidb-binlog-configuration-file.md:352]: 是的

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 5]
```diff
@@ -334,22 +334,19 @@ tbl-name = "~^a.*"
 
 ### syncer.to.checkpoint
 
-以下是 `syncer.to.checkpoint` 相关的配置项。
+* `type`：指定用哪种方式保存同步进度，目前支持的选项为 `mysql`、`tidb` 和 `file`。
 
-### type
+> **注意：**
+> file 类型的下游 checkpoint 保存在 `<data-dir>/savepoint` 文件中
+> 当 `mysql` 或 `tidb` 类型保存同步进度时，需要指定以下配置项：
```
Comment (TomShawn) [inline tidb-binlog/tidb-binlog-configuration-file.md:341]:
```suggestion

    该配置选项默认与下游类型相同。例如 `file` 类型的下游 checkpoint 进度保存在本地文件 `<data-dir>/savepoint` 中，`mysql` 类型的下游进度保存在下游数据库。当明确指定要使用 `mysql` 或 `tidb` 保存同步进度时，需要指定以下配置项：
```

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 6]
```diff
@@ -334,22 +334,19 @@ tbl-name = "~^a.*"
...
 
-* 指定用哪种方式保存同步进度。
-* 目前支持的选项：`mysql` 和 `tidb`
+* `schema`：默认为 `"tidb_binlog"`。
 
-* 默认：与下游类型相同。例如 `file` 类型的下游进度保存在本地文件系统，`mysql` 类型的下游进度保存在下游数据库。当明确指定要使用 `mysql` 或 `tidb` 保存同步进度时，需要指定以下配置项：
+  > **注意：**
+  >
+  > 在同个 TiDB 集群中部署多个 Drainer 时，需要为每个 Drainer 节点指定不同的 checkpoint schema，否则两个实例的同步进度会互相覆盖。
```
Comment (TomShawn) [inline tidb-binlog/tidb-binlog-configuration-file.md:347]:
```suggestion
      > **注意：**
      >
      > 在同个 TiDB 集群中部署多个 Drainer 时，需要为每个 Drainer 节点指定不同的 checkpoint schema，否则两个实例的同步进度会互相覆盖。
```

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 7]
```diff
@@ -334,22 +334,17 @@ tbl-name = "~^a.*"
...
+    该配置选项默认与下游类型相同。例如 `file` 类型的下游 checkpoint 进度保存在本地文件 `<data-dir>/savepoint` 中，`mysql` 类型的下游进度保存在下游数据库。当明确指定要使用 `mysql` 或 `tidb` 保存同步进度时，需要指定以下配置项：
 
-* 指定用哪种方式保存同步进度。
-* 目前支持的选项：`mysql` 和 `tidb`
+* `schema`：默认为 `"tidb_binlog"`。
 
-* 默认：与下游类型相同。例如 `file` 类型的下游进度保存在本地文件系统，`mysql` 类型的下游进度保存在下游数据库。当明确指定要使用 `mysql` 或 `tidb` 保存同步进度时，需要指定以下配置项：
+      > **注意：**
+      >
+      > 在同个 TiDB 集群中部署多个 Drainer 时，需要为每个 Drainer 节点指定不同的 checkpoint schema，否则两个实例的同步进度会互相覆盖。
```
Comment (TomShawn) [inline tidb-binlog/tidb-binlog-configuration-file.md:345]:
```suggestion
    > **注意：**
    >
    > 在同个 TiDB 集群中部署多个 Drainer 时，需要为每个 Drainer 节点指定不同的 checkpoint schema，否则两个实例的同步进度会互相覆盖。
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**